src/ui/note/mod.rs

```diff
@@ -528,19 +529,22 @@ fn get_reposted_note<'a>(ndb: &Ndb, txn: &'a Transaction, note: &Note) -> Option
 }
 
 fn note_hitbox_id(note_key: NoteKey) -> egui::Id {
-    Id::new(("note_rect", note_key))
+    Id::new(("note_size", note_key))
```

Comment:
note size may be different depending on the context. for example, they would be smaller in quoted notes, so we should be keying on that context. something to keep in mind.

since our columns are the same width, this shouldn't really change between columns, but we may need to update this for quoted notes at least to include context information like hashed note options.